Content.Server/Spider/SpiderSystem.cs

```diff
@@ -0,0 +1,78 @@
+using System.Linq;
+using Content.Server.Popups;
+using Content.Shared.Spider;
+using Content.Shared.Maps;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map;
+
+namespace Content.Server.Spider
+{
+    public sealed class SpiderSystem : EntitySystem
+    {
+        [Dependency] private readonly PopupSystem _popup = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<SpiderComponent, SpiderWebActionEvent>(OnSpawnNet);
+        }
```

Comment:
Inherit `SharedSpiderSystem`  
Im pretty sure you can just subscribe the `SpiderWebActionEvent` there then handle it here, tho i dont remember if actions had some quirkiness with that so disregard this last bit if it doesn't work